src/jvm.jl

```diff
@@ -312,17 +298,16 @@ function _init(opts)
     opt = [JavaVMOption(pointer(x), C_NULL) for x in opts]
     ppjvm = Array{Ptr{JavaVM}}(undef, 1)
     ppenv = Array{Ptr{JNIEnv}}(undef, 1)
-    vm_args = JavaVMInitArgs(JNI_VERSION_1_6, convert(Cint, length(opts)),
+    vm_args = JavaVMInitArgs(JNI.JNI_VERSION_1_8, convert(Cint, length(opts)),
```

Comment:
What's our minimum JVM version with this? Should probably document it?

Reply:
This would go along with Java 8, although as far as I can tell there really is not a difference between 1_6 and 1_8 in terms of the JNI, so maybe we should revert this.

Reply:
This is documented here: https://docs.oracle.com/javase/9/docs/specs/jni/functions.html

As you previously noted, this correspondence is missing here in the Java 8 notes:
https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#GetVersion